packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -154,12 +154,26 @@ function promptBeforeLaunchingWorkspace(
  * @param name The name of the workspace to launch
  * @param additionalProps Props to pass to the workspace component being launched
  */
-export function launchPatientWorkspace(name: string, additionalProps?: object) {
+
+const myFunction = () => {
+  const orderBasketOpen = true;
+  if (orderBasketOpen) {
+    navigate({ to: '/order-basket' });
+  }
+};
+
+launchPatientWorkspace('drug-search-workspace', {
+  onClosingWorkspace: myFunction,
+});
```

Comment:
Hi @jwnasambu , as we discussed in the Coffee break, this namely /`myFunction` will be coming from `launchPatientWorksapce('drug-order-workspace')` and will not be present here.

Reply:
@vasharma05 Thanks so much for the timely guidance. I really needed it you are such a blessing. Kindly let me embark on it tomorrow after the sabbath!